src/state/inventory.ts

```diff
@@ -73,25 +73,61 @@ export class Inventory {
             });
     }
 
-    private loadServices (): Service[] {
+    private get __servicesCacheObject (): Service[] {
```

Comment:
Do we really need the `__` prefix? Typescript has access modifiers which renders them unnecessary, they're needed in languages such as Python without access modifiers but we don't need them really within a Typescript class.

Reply:
Meant for it to be a strong indication of "Do not touch this unless you know what you’re doing." private function.